docs/components/drawer/index.html

```diff
@@ -0,0 +1,73 @@
+---
+title: Drawer
+minver: 0.17.0
+also:
+  components/drawer/test.html: Testing - Drawer
+  elements/hx-drawer: <hx-drawer>
+---
+{% extends 'component.njk' %}
+
+{% block page_header %}
+  <p>
+    <code>&lt;hx-drawer&gt;</code> provides a temporary overlay panel
+    that expedites the completion of tasks related to the user's current
+    context.
+  </p>
+{% endblock %}
+
+{% block content %}
+    <section>
+      <header>
+        <h2 id="basic-drawer">Basic Drawer</h2>
+      </header>
+
+      <div class="example" id="vue-drawerDemo" v-cloak>
+        <div>
+          <hx-disclosure
+            aria-controls="demo-drawer"
+            class="hxBtn"
+            id="open-drawer"
+          >
+            Open Drawer
+          </hx-disclosure>
+
+          <hx-drawer
+            id="demo-drawer"
+          >
+            <header>
+              <h3>Basic Drawer</h3>
+            </header>
+
+            <hx-div class="hxMd">
+              <p>
+                This is the body of the drawer with padding added.
+              </p>
+            </hx-div>
+
+            <footer>
+              <button class="hxBtn hxPrimary">Confirm</button>
+              <button class="hxBtn">Cancel</button>
+            </footer>
+          </hx-drawer>
+        </div>
+
+        <footer>
+          <pre><code v-text="snippet"></code></pre>
+        </footer>
+      </div>
+
+      <footer>
+        <p class="hxSubdued hxSubBody">
+          <hx-icon type="info-circle"></hx-icon>
+          <code>&lt;hx-drawer&gt;</code> content does not include padding by default.
```

Comment:
update to reference `hx-div` and not the drawer

Reply:
Wording updated to reference adding padding to `<hx-div>`.